### DISCUSSION TITLE
yiwen-blog-website/docs/native/idea_search

### DISCUSSION THREAD

**Discussion by giscus** [Hao-yiwen/yiwen-blog-website#255] [Announcements]
# yiwen-blog-website/docs/native/idea_search

so 文件和 a 文件都是用于共享代码库的文件格式，但它们在使用和行为上有显著的区别。

https://hao-yiwen.github.io/yiwen-blog-website/docs/native/idea_search

**Comment by Hao-yiwen** [comment]
command+e是最近的文件快捷键，这也是一个很有用的快捷键。
